### ISSUE TITLE
Pretrain confusion on Voxceleb

### ISSUE THREAD

**Issue by Huangzhw0221** [sunlicai/MAE-DFER#4]
Great work! But I'm getting some confusion when pretrain on Voxceleb2 with raw VideoMae and VideoMae2. 
1. Both of the two model met '"train_grad_norm": Infinity' when pretraining, while the training losses are really declining. Is there some explanation to illustrate this, or is there some methods to avoid this problem? Have you met this problem when pretraining?
2. I tried pretrain VideoMae on MAFW and also finetune on MAFW with train set, however the acc only reaches 25%. So is this caused by a violation or is there some other reason? The accuracy of pre-training and fine-tuning and 101 classification on the UCF101 dataset is quite good, and the video number of MAFW is close to UCF101.
3. I venture to ask if there is a chance to see the run-pretrain code and pretrain logs for MAE-DFER in the future.
Good luck with your research!

**Comment by youcaiSUN** [comment]
Hi,  @Huangzhw0221 ! Thanks for your interest in our work!

1. MAE-DFER pre-trained on VoxCeleb2 also got "train_grad_norm": Infinity. I am also confused with this problem. Since our code is based on VideoMAE, I encourage you to explore in their repo for the possible answer and I'd love to hear from you if you find that.
2. I did not pre-trained the model on MAFW because the main goal of this paper is to investigate the feasiblity of  large-scale self-supervised pre-training on *unlabeled* data to DFER. You can try to pre-train VideoMAE longer (e.g., 10000 epochs) on MAFW since I verified that the model performed not bad (about 70\% WAR on DFEW) when only pre-trained on a VoxCeleb2 subset which has only 11K video clips. 
3. I've uploaded the pre-training log [here](https://github.com/sunlicai/MAE-DFER/blob/master/saved/model/pretraining/voxceleb2/videomae_pretrain_base_dim512_local_global_attn_depth16_region_size2510_patch16_160_frame_16x4_tube_mask_ratio_0.9_e100_with_diff_target_server170/log_50_epochs.txt). For the pre-training code, I'll refactor it and upload it in early December. Stay tuned!

Best,
Licai
